cli/src/commands/version.rs

```diff
@@ -63,6 +70,7 @@ Report bugs: <https://github.com/martinvonz/jj/issues>
     if args.build_info {
         writeln!(ui.stdout())?;
         writeln!(ui.stdout(), "Target: {}", env!("JJ_CARGO_TARGET"))?;
+        writeln!(ui.stdout(), "Commit: {}", git_commit)?;
```

Comment:
Do we want to allow redistributors the option of adding another suffix here, so it can be even clearer how it was built?

e.g. my `rustc version` gives `rustc 1.78.0 (9b00956e5 2024-04-29) (Alpine Linux 1.78.0-r0)`

Reply:
Yes, and I think you should also be able to set some information through the API, too.